src/averageAges.js

```diff
@@ -15,11 +15,24 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = people.filter(person => century
+    ? (genderIs(person, 'm') && fromCentury(person, century))
+    : (genderIs(person, 'm')));
+
+  return avarageAge(men);
+}
+
+function genderIs(person, sex) {
+  return person.sex === sex;
+}
+
+function fromCentury(person, century) {
+  return Math.ceil(person.died / 100) === century;
+}
+
+function avarageAge(people) {
```

Comment:
- misspelling
- functions should usually start with a verb, so it was obvious that we're dealing with a function
```suggestion
function getAverageAge(people) {
```